src/pages/clients/common/components/CustomResourcefulActions.tsx

```diff
@@ -46,6 +46,7 @@ export function CustomResourcefulActions(props: Props) {
           to={route('/clients/:id/statement', {
             id: props.clientId,
           })}
+          iconType="view"
```

Comment:
I don't think I get this. What does icon type "new" has to do with "View statement"?

Reply:
Hmm, I think that you missed it. It's actually, 'view'.